legacy-pages/js/import/wallet-import.js

```diff
@@ -236,6 +250,21 @@
         evt.dataTransfer.dropEffect = 'copy'; // Explicitly show this is a copy.
     }
 
+    function initCaptcha() {
+        if (!window.grecaptcha || !window.grecaptcha.enterprise) return
+        window.grecaptcha.enterprise.ready(() => {
+            window.grecaptcha.enterprise
+                .execute(window.CAPTCHA_KEY, { action: 'IMPORT_JSON_WALLET' })
+                .then((captchaToken) => {
+                    console.log('Captcha success', captchaToken)
+                    window.NEWRECAPCHA = captchaToken
```

Comment:
Not sure about using a global variable for this.

Thanks @schnogz [for the reference](https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4-frontend/src/scenes/Signup/index.tsx#L47-L63)